src/types.ts

```diff
@@ -123,6 +123,7 @@ export interface SamlOptions extends Partial<SamlSigningOptions>, MandatorySamlO
 
   // extras
   disableRequestAcsUrl: boolean;
+  samlExtensions?: any;
```

Comment:
Should be of type `object`.

Reply:
`@typescript-eslint/ban-types` does not allow the type `object`.
```
Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys.eslint@typescript-eslint/ban-types
```

Reply:
Sure, use that instead. That will help TypeScript users make sure they get at least the basics of what is supposed to go in this field.

Reply:
Sure, Done